.github/workflows/post-merge-matrix-deploy.yml

```diff
@@ -41,52 +52,64 @@ jobs:
       id-token: write
       contents: read
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout code
+        if: matrix.ENABLED == 'true'
```

Comment:
Are you able to have this conditional one level above, or does that skip the entire matrix?

Reply:
No can't have it above as it is not defined at that point, I did try a few variations on it but got invalid workflow syntax, GitHub is a bit lacking in this area